include/deal.II/lac/trilinos_solver.h

```diff
@@ -109,19 +106,18 @@ namespace TrilinosWrappers
     /**
      * Constructor. Takes the solver control object and creates the solver.
      */
-    SolverBase (SolverControl  &cn);
+    SolverBase (SolverControl &cn);
 
     /**
      * Second constructor. This constructor takes an enum object that
      * specifies the solver name and sets the appropriate Krylov method.
      */
-    SolverBase (const enum SolverName  solver_name,
-                SolverControl         &cn);
+    SolverBase (const enum SolverName solver_name, SolverControl &cn);
 
     /**
      * Destructor.
      */
-    virtual ~SolverBase ();
+    virtual ~ SolverBase ();
```

Comment:
..and here, and multiple other places -- please go through the patch and make sure that you don't change things that don't need changing.